Gemfile

```diff
@@ -28,6 +28,9 @@ gem 'ri_cal'
 gem 'rails_autolink'
 gem 'faker', require: false
 
+gem 'pry-rails'
+gem 'pry-byebug', require: false
```

Comment:
Why did you not *fully* switch over to `pry`? You could have removed `byebug` in the test and dev and instead throw in `pry-byebug` there. Or do you see a usage for `byebug`there still?